magprime/templates/emails/precon_faqs.html

```diff
@@ -8,23 +8,24 @@
 <br/> <br/>
 
 <p><b> Will passes be sold at the door?</b><br />
-Yes. Full four-day passes will be available for $70, and one day passes will be available for $35 for Friday, $40 for Saturday, $40 for Sunday, and $30 for Monday.</p>
+Yes, but there is a possibility of us selling out, so check our website for any ticket availability alerts. Full four-day passes will be available for $80, and one day passes will be available for $40 for Thursday, $45 for Friday, $45 for Saturday, and $25 for Sunday.</p>
```

Comment:
Silly question: this email is by definition being sent to people who have already preregistered and have a badge waiting for them when they arrive.  Does it make sense for the very first question to be about how they can purchase a badge?